scripts/topiary

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Yea, works for me since I mainly use 3, but I think most still don't. Many academic cluster users won't have much of a choice on this and the netmhc tools require py2 so you'd need both installed.